src/routes/news/components/NewsListItem.js

```diff
@@ -0,0 +1,158 @@
+// @flow
+
+import * as React from 'react'
+import {
+  LocalNewsModel,
+  TunewsModel
+} from '@integreat-app/integreat-api-client'
+import styled from 'styled-components/native'
+import { type TFunction, withTranslation } from 'react-i18next'
+import type { ThemeType } from '../../../modules/theme/constants/theme'
+import { type StyledComponent } from 'styled-components'
+import {
+  contentDirection,
+  contentAlignment
+} from '../../../modules/i18n/contentDirection'
+
+type PropsType = {|
+  newsItem: LocalNewsModel | TunewsModel,
+  language: string,
+  navigateToNews: () => void,
+  theme: ThemeType,
+  t: TFunction,
+  isTunews: boolean
+|}
+
+type ListItemViewPropsType = {|
+  language: string,
+  children: React.Node,
+  theme: ThemeType
+|}
+
+const ListItemView: StyledComponent<
+  ListItemViewPropsType,
+  ThemeType,
+  *
+> = styled.View`
+  flex-direction: ${props => contentDirection(props.language)};
+`
+
+const ReadMoreWrapper: StyledComponent<
+  { language: string, children: React.Node },
+  {},
+  *
+> = styled.View`
+  flex-direction: ${props => contentDirection(props.language)};
+  justify-content: flex-end;
+  width: 100%;
+`
+
+const ListItemWrapper = styled.View`
+  padding-horizontal: 5%;
+`
+
+const StyledTouchableOpacity: StyledComponent<
+  {},
+  ThemeType,
+  *
+> = styled.TouchableOpacity`
+  flex-direction: column;
+`
+
+const Divider: StyledComponent<{}, {}, *> = styled.View`
+  border-top-width: 0.5px;
+  border-top-color: rgba(168, 168, 168, 0.7);
+  width: 80%;
+  margin-top: 12px;
+  margin-bottom: 12px;
+  align-self: center;
+`
+
+export const Description: StyledComponent<
+  { isTunews: ?boolean },
+  ThemeType,
+  *
+> = styled.View`
+  flex-direction: column;
+  font-family: ${props => props.theme.fonts.decorativeFontRegular};
+  padding: ${props => (props.isTunews ? '15px 5px 0px 0px' : '0px')};
```

Comment:
Why is this only needed for tünews?

Reply:
cleaned. It's was because there was different styles for both types but i think no need for it now.